packit_service/worker/handlers/github_handlers.py

```diff
@@ -273,6 +273,7 @@ def run(self) -> TaskResults:
                 self.copr_build_helper.report_status_to_all(
                     description=PERMISSIONS_ERROR_WRITE_OR_ADMIN,
                     state=CommitStatus.failure,
+                    url=FAQ_URL,
```

Comment:
I can't find anything about that situation on the FAQ page.

Reply:
The direct link to the relevant part is: https://packit.dev/packit-as-a-service/#how-to-re-trigger-packit-service-actions-in-your-pull-request

Maybe we should use that instead? The FAQ page is indeed long and users might also struggle with finding the relevant section.

At the same time, might worth to make that sentence in the FAQ about the permissions a little bit more visible.